### DISCUSSION TITLE
Send in the future Option

### DISCUSSION THREAD

**Discussion by githubgigmasters** [0perationPrivacy/VoIP#145] [Ideas]
One thing I love about my default Android Text Messenger App is I can set a SMS to be sent a future date and time. Something similar would be nice to see in the future builds. The ability for a message to get sent to the server and then the server hold on to it until the time comes to release the message or something like that.

**Comment by 0perationPrivacy** [comment]
interesting option, can be implemented, I'll put it in the pipeline for a future build. 👍🏻 thanks
Issue https://github.com/0perationPrivacy/VoIP/issues/146
